### PR TITLE
Added/modified latest sudo version available in portage

### PR DESCRIPTION
**Pull request by cslycord**
Added the most recent version of app-admin/sudo and modified it to use libressl.

This time, I remembered to included Manifests :)

### CODE REVIEW

**Comment by cslycord** [comment]
Latest sudo from portage modified to work w/ libressl.

File: app-admin/sudo/Manifest [hunk 1]
```diff
@@ -1,3 +1,5 @@
...
 DIST sudo-1.8.23.tar.gz 3150674 BLAKE2B 11b1c7bfa372005cda8baf651c4662f6fd15e94ca77f7705b23ca6573424796d5c1f8e47e2874c4b54017141d01a632885ac60c92346d932537048373cad0ede SHA512 a9d61850a4857bfd075547a13efb13b054e4736e3ebe3c8a98a90a090b1d9b9688354ec9725fc99d1d256999b6f9c6ae6215ce9770fcdebd7f24731107b48342
+DIST sudo-1.8.25p1.tar.gz 3189951 BLAKE2B ebfedaad62e60f625db8c46a5c8f19977a5ec0a86bab3b34d91096c08e8b8ece056ba312f9fecd4cdd704fc17d49a36681b41cd40269df7c67cd66d80c0d8efb SHA512 b1445be688d3c1dd7efbdfab68977a7a9b6fd6887191dc99ca717117eec0a550492642556cd55ca5873d054ddc5ccc2b87b2c34602e1ffc729ab6fbc4e523a72
+EBUILD sudo-1.8.25_p1-r1.ebuild 6530 BLAKE2B 522ae455b21dc5d1435826a518790fb1bacfa43a6edd958c8a9c0d0e2ef9dd2588f82ff4b9b8f89801b7e24b622aa84c72e3a173110c8724edf11bcdce9385fd SHA512 10f03f5986fcab0087bb4e0e968b231bc13a7a2c0af14935fb4cf21ae67e8952dc7338dfe90f9ac38bc89cad12978ac259fe2012fd0023214df5b1106f1ea73e
```
Comment (ian-moone) [inline app-admin/sudo/Manifest]: Please remove the digests for `EBUILD`. There's no need for it, as we use [`thin-manifests`](https://github.com/gentoo/libressl/blob/master/metadata/layout.conf#L2).

File: app-admin/sudo/sudo-1.8.25_p1-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+	if [[ ${PV} != *_beta* ]] && [[ ${PV} != *_rc* ]] ; then
+		KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~sparc-solaris"
+	fi
+fi
+
+# Basic license is ISC-style as-is, some files are released under
+# 3-clause BSD license
+LICENSE="ISC BSD"
+SLOT="0"
+IUSE="gcrypt ldap libressl nls offensive openssl pam sasl selinux +sendmail skey sssd"
```
Comment (ian-moone) [inline app-admin/sudo/sudo-1.8.25_p1-r1.ebuild]: `sssd` is a new USE flag. Please update `metadata.xml` accordingly. 👉 https://travis-ci.org/gentoo/libressl/builds/444330684#L537

Comment (cslycord) [reply app-admin/sudo/sudo-1.8.25_p1-r1.ebuild]: Thanks for the heads-up on that, by the way. I wasn't aware that it'd been changed with that revision number.

**Pull request merged**